.github/workflows/backfill-corr-ci.yml

```diff
@@ -10,57 +10,37 @@ name: R backfill corrections
 
 on:
   push:
-    branches: [ main, prod ]
+    branches: [main, prod]
   pull_request:
-    types: [ opened, synchronize, reopened, ready_for_review ]
-    branches: [ main, prod ]
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches: [main, prod]
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     if: github.event.pull_request.draft == false
-    strategy:
-      matrix:
-        r-version: [4.2.1]
     defaults:
       run:
         working-directory: backfill_corrections/delphiBackfillCorrection
 
     steps:
-      - uses: actions/checkout@v2
-      - name: Set up R ${{ matrix.r-version }}
+      - uses: actions/checkout@v4
+
+      - name: Set up R 4.2
         uses: r-lib/actions/setup-r@v2
         with:
-          r-version: ${{ matrix.r-version }}
           use-public-rspm: true
-      - name: Install linux dependencies
-        run: |
-            sudo apt-get install \
```

Comment:
question: we don't need this section because `r-lib/actions/setup-r-dependencies@v2` installs using `pak`, which as you mentioned above also installs dependencies?